src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/config/MongoConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.itemgroupworkflowapi.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.MongoDatabaseFactory;
+import org.springframework.data.mongodb.core.convert.DefaultDbRefResolver;
+import org.springframework.data.mongodb.core.convert.DefaultMongoTypeMapper;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
+/**
+ * Custom configuration for Mongo added so that _class attributes are not saved on objects stored in MongoDB.
+ * Taken from uk.gov.companieshouse.certificates.orders.api.config => MongoConfig.java
+ * NOTE custom conversions String->enum and enum->String are NOT included.
+ */
+@Configuration
+public class MongoConfig {
+    /**
+     * _class maps to the model class in mongoDB (e.g. _class : uk.gov.companieshouse.item.group.workflow.api.model.ItemGroup)
```

Comment:
The package of the `ItemGroup` class referred to has changed. If we use a `@link`, tools like Idea will point out these breaks to us.